_includes/live-demos/drive-standalone-browse-controls/example.js

```diff
@@ -0,0 +1,17 @@
+var button = document.querySelector('.tinydrive-standalone-demo-browse');
+
+button.addEventListener('click', function () {
+  var controls = tinydrive.browse({
+    token_provider: '/your-local/jwt-provider'
+  }, function() {
+    setTimeout(function() {
```

Comment:
Should we have a comment here to highlight this just simulates closing the picker manually at a later stage?

Reply:
That's not what it's there for, it's there to prevent closing the picker before the browse-function has returned the control-object.

Reply:
I left a comment elsewhere, but this isn't what I was lead to believe. What was said previously is that this new API is designed to allow people to close the dialog at a later stage when they need to, so that what we should be showing here.

If it actually needs to set the timeout to safely work, then that's not an acceptable implementation and it needs to be changed sorry.